### PR TITLE
github page: let a reporter to use makrdown notation for pre-formatti…

### PR DESCRIPTION
**Pull request by masatake**
…ng in the issue report interface

Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

**Comment by masatake** [comment]
@b4n, thank you very much.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,73 @@
-(If you will report an issue about the result of parsing,
-fill the following items. If you get the input file somewhere,
-write the URL or something.)
+(
+Thank you for contacting us.
+
+*If you are reporting an issue with the parsing output, please fill
+the following template.  As your custom CTags configuration can
+affect results, please always use `--options=NONE` as the first
+option when running `ctags`.*
```
Comment (b4n) [inline .github/ISSUE_TEMPLATE.md]: I'd remove the italics (`*` around), I had them in my comment only to highlight the part that I suggested as replacement, sorry.

Comment (masatake) [reply .github/ISSUE_TEMPLATE.md]: Oh, I see.

**Comment by masatake** [comment]
Thank you.

**Pull request merged**